_works/2025-maybauer-finn/02-titles-and-research-questions.md

```diff
@@ -0,0 +1,35 @@
+# Titel und Forschungsfrage
+
+Bitte notieren Sie mindestens 10 mögliche Titel der Arbeit und mindestens 5 mögliche Forschungsfragen/Zielsetzungen in dieses Dokument.
+
+## Titel
+
+### Iteration 0
+
+> [!NOTE]  
+> Titel generiert über ChatGPT 4o. Prompt: "Generiere 10 mögliche Titel für eine Masterarbeit in der Medieninformatik mit folgendem Expose: <[Inhalt des Exposes](03-expose.md)"
+
+* Barrierefreiheit in Unternehmen: Ein Reifegrad-Ansatz zur Umsetzung der EU-Norm EN 301 549
+* Von Compliance zu Inklusion: Strategien zur Messung und Steigerung der Barrierefreiheitsreife
+* Reifegradmodelle für barrierefreie digitale Produkte: Herausforderungen und Chancen
+* Barrierefreiheit als Prozess: Entwicklung eines Werkzeuges zur Maturity-Messung
+* Inklusive Digitalisierung: Methoden zur Steigerung der Barrierefreiheit in Organisationen
+* Workshop oder Web-Tool? Ansätze zur Prozessoptimierung in der Barrierefreiheit
+* Maturity-Modell-Ansätze für die Barrierefreiheit: Eine Fallstudie
+* Barrierefreiheits-Compliance: Werkzeuge und Methoden zur organisatorischen Vorbereitung
+* Messbare Inklusion: Ein Reifegradmodell zur Umsetzung der EU-Norm EN 301 549
+* Digitale Barrierefreiheit im Fokus: Entwicklung einer Methodik zur Prozessverbesserung
+
+### Iteration 1
+
```

Comment:
Grundsätzlich wäre die Frage, ob der Begriff "Barrierefreiheit" nicht zu umfassend ist. Gehst Du in Richtung Digitale Barrierefreiheit? Deine Map sagt: ja.

Reply:
Grundsätzlich würd ich sagen ja. Auf der einen Seite behandelt die EU-Norm nicht nur den digitalen Raum. Auf der anderen Seite sind viele Maturity-Modelle speziell für digitale Barrierefreiheit entwickelt.

Spannend ist es, wenn Maturity-Modelle auch die Arbeitsweise im Unternehmen Barrierefrei betrachtet (Inklusion am Arbeitsplatz oder in der Map "Zugängliches Arbeiten", "heterogenes Team" => Wäre dann auch wieder was für die Führungsebene).

Wir müssten da glaub ich nochmal darüber in der Sprechstunde diskutieren, wo wir den Cut setzen.